okhttp3/src/main/java/zipkin2/reporter/okhttp3/Platform.java

```diff
@@ -35,18 +35,31 @@ static Platform get() {
   static Platform findPlatform() {
     // Find JRE 8 new types
     try {
-      Class.forName("java.io.UncheckedIOException");
-      return new Jre8(); // intentionally doesn't not access the type prior to the above guard
+      Class<?> clazz = Class.forName("java.io.UncheckedIOException");
+      Constructor<?> ctor = clazz.getConstructor(IOException.class);
```

Comment:
this is annoying, but using this path is also very unlikely. It meant there was an IOException gzipping spans, which would be uncommon for sure.